includes/class-wc-calypso-bridge-setup.php

```diff
@@ -680,6 +686,46 @@ public function prevent_redirects_on_activation( $location, $status ) {
 	private function write_to_log( $operation, $message ) {
 		error_log(  'WooExpress: Operation: (' . microtime( true ) . ') ' . $operation . ': ' . print_r( $message, 1 ) );
 	}
+
+	/**
+	 * Delete page by slug
+	 *
+	 * @param string $slug Slug.
+	 * @param string $operation Operation.
+	 *
+	 * @since x.x.x
+	 *
+	 * @return void
+	 */
+	private function delete_page_by_slug( $slug, $operation ) {
+
+		$page = get_page_by_path( $slug, ARRAY_A );
+
+		if ( is_array( $page ) && isset( $page['ID'] ) ) {
```

Comment:
Totally minor, non-blocking suggestion: this might be a bit more readable as an early return.
```suggestion
		if ( ! is_array( $page ) || ! isset( $page['ID'] ) ) {
			return;
		}
```

Reply:
I'm all in for readability and bail out early! Thanks!